Still need to download source (#2)
ctron/iot-simulator-example: README.md

There are two ways to Deploy enmasse to your cluster, either **manually** or via the provided [**Enmasse operator**](https://operatorhub.io/operator/enmasse)

> Reguardless please download and install the enmasse Tarball since we do some setup with these via the commandline

Reviewer: ```suggestion
Regardless please download and install the EnMasse tarball since we do some setup with these via the command line
```

### Manually

Make sure you are on the correct openshift project 

>    `[astoycos@localhost iot-simulator-example]$ oc project enmasse-infra

Reviewer: ```suggestion
    $ oc project enmasse-infra
```

>    Now using project "enmasse-infra" on server "https://api.astoycos-ocp.shiftstack.com:6443".`

Reviewer: ```suggestion
    Now using project "enmasse-infra" on server "https://api.my-cluster.my-domain:6443".`
```
